cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,62 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  confirmTheProduct(productName) {
+    cy.get('#tbodyid')
+      .should('contain', productName);
+  }
+
+  clickOnOrder() {
+    cy.contains('.btn', 'Place Order')
+      .click();
+  }
+
+  typeName(name) {
+    cy.get('#name')
+      .type(name, { force: true });
+  }
+
+  typeCountry(country) {
+    cy.get('#country')
+      .type(country, { force: true });
+  }
+
+  typeCity(city) {
+    cy.get('#city')
+      .type(city, { force: true });
+  }
+
+  typeCardnumder(cardnumder) {
+    cy.get('#card')
+      .type(cardnumder, { force: true });
+  }
+
+  typeMonth(month) {
+    cy.get('#month')
+      .type(month, { force: true });
+  }
+
+  typeYear(year) {
+    cy.get('#year')
+      .type(year, { force: true });
+  }
+
+  clickOnPurchase() {
+    cy.contains('.btn', 'Purchase')
+      .click();
+  }
```

Comment:
you could create one universal function for buttons and pass the name of a button as an argument